### ISSUE TITLE
Weapon Bob Speed - Too Slow?

### ISSUE THREAD

**Issue by montgocloud** [shinyquagsire23/OpenJKDF2#339]
The weapon bob seems to be much slower than it should be - or at least much slower than I'm used to? AFAIK it's supposed to be ~2 steps per bob (2 steps -> bob right, 2 steps -> bob left, etc.), but here it's more like... 8 or 10 steps. 

https://youtu.be/qWuRQYIa-d4 
This is on a completely fresh install of the game with no mods.

**Comment by Calinou** [comment]
I agree it doesn't make a lot of physical sense, but it's like in Doom where the bobbing is pretty slow but it has a high amplitude to compensate.

If bobbing was made to look faster, you'd want the amplitude to be smaller as to not be too distracting. It could make for an interesting option to have.

**Comment by montgocloud** [comment]
I guess I'm more concerned about accuracy to the original game - was it this slow at original release and the faster pace I'm used to is a visual bug? I don't have any indication of that, but if that's the case then that's fine.

**Comment by shinyquagsire23** [comment]
Improvement staged for v0.9.5, I set the waggle speed to be the same as it was at 25fps (which is my vague hand-wavey assumption at the FPS they had when coding the waggle)
